### ISSUE TITLE
My code stays at the tqdm 

### ISSUE THREAD

**Issue by mytk2012** [zylo117/Yet-Another-EfficientDet-Pytorch#433]
details are as bellows:loaded weights: efficientdet-d0.pth, resuming checkpoint from step: 0
0%|                                                     | 0/6 [00:00<?, ?it/s]
and when i debug with pycharm，i find it always stay at :cls_loss, reg_loss = model(imgs, annot, obj_list=params.obj_list)。How should i handle with this？

**Comment by mytk2012** [comment]
notably，my 4 gpus run all the time

**Comment by zylo117** [comment]
try debugging with cpu by setting GPUS: 0, and num_workers = 0

**Comment by mytk2012** [comment]
> try debugging with cpu by setting GPUS: 0, and num_workers = 0
It succeded when using CPU，but if with GPU，what i should do？

**Comment by mytk2012** [comment]
> try debugging with cpu by setting GPUS: 0, and num_workers = 0

Can you give me some suggestions why my cls loss is very large about 44000，though it decreases to 2251 after 119 epochs

**Comment by mytk2012** [comment]
> try debugging with cpu by setting GPUS: 0, and num_workers = 0

And Reg loss remains 0 all the time.

**Comment by mytk2012** [comment]
Are there some problems with my own datasets？But I have replaced mean and std values in coco.yml

**Comment by zylo117** [comment]
what is the mean and std now?

**Comment by zylo117** [comment]
loss is large at first but it decreases very fast.
And after 119 epochs, it is still 2251? I think there is something wrong with your setting or the dataset. You should refer to the tutorial.

**Comment by mytk2012** [comment]
> loss is large at first but it decreases very fast.
> And after 119 epochs, it is still 2251? I think there is something wrong with your setting or the dataset. You should refer to the tutorial.

I am sorry to interrupt you.I have succeeded by changing datasets and labels.There are too many differences among my datasets.

**Comment by rohanbanerjee** [comment]
Hey @zylo117 
Thank you for the amazing script

I was also kind of facing the same issue here, the training isn't proceeding beyond "Initializing weights...". I have assigned three GPUs to it, all of them are being used but the training isn't proceeding. Any kind heads up would be really helpful

Thanks!
